src/pages/home/report/ReportActionsList.js

```diff
@@ -30,13 +30,7 @@ const propTypes = {
     report: reportPropTypes.isRequired,
 
     /** Sorted actions prepared for display */
-    sortedReportActions: PropTypes.arrayOf(PropTypes.shape({
-        /** Index of the action in the array */
-        index: PropTypes.number,
```

Comment:
`index` isn't needed anymore? I see it used in `renderItem` but maybe the update `item: reportAction,` takes care of this? Just curious

Reply:
Yeah, this isn't needed anymore. The `index` you see in `renderItem` comes from React Native itself.

Reply:
Thanks for the tip